File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/Address.kt
package com.br.app.deliverydelicatessen.model

data class Address(
    var id: Int = 0,
    var establishmentId: Int? = 0,
    var street: String = "",
                   var reference: String = "",
                   var postalCode: String = "",
                   var district: String = "",
                    var city: String = "",
                   var uf: String = ""){
    constructor():this(0,null,"","","","", "", "")

}

File: financial/Models/Address.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Address")]
    public class Address : BaseEntity
    {
        public string ApplicationUserId { get; set; }
        public string Street { get; set; }
        public string Reference { get; set; }
        public string PostalCode { get; set; }
        public string District { get; set; }
        public string City { get; set; }
        public string Uf { get; set; }

        [NotMapped]
        public int EstablishmentId { get; set; }

    }
}
